### PR TITLE
Added a new function call to the Color class: ToHtml()

### PR DESCRIPTION
**Pull request by mee-ironsoftware**
This translates the specified Color structure to an HTML string color representation.

### CODE REVIEW

**Review by michael-ironsoftware — APPROVED** [review]
LGTM 👍

File: IronSoftware.Drawing/IronSoftware.Drawing.Common/Color.cs [hunk 1]
```diff
@@ -1251,6 +1251,15 @@ public override int GetHashCode()
             return 0;
         }
 
+        /// <summary>
+        /// Translates the specified Color structure to an HTML string color representation.
+        /// </summary>
+        /// <returns>A string containing the hex representation of the color in the format #RRGGBB.</returns>
+        public string ToHtml()
```
Comment (iron-production-team) [inline IronSoftware.Drawing/IronSoftware.Drawing.Common/Color.cs:1258]: ToHtmlCssColorCode

Comment (iron-production-team) [reply IronSoftware.Drawing/IronSoftware.Drawing.Common/Color.cs:1258]: or TranslateToHtmlColor()

Comment (mee-ironsoftware) [reply IronSoftware.Drawing/IronSoftware.Drawing.Common/Color.cs]: Changing from ToHtml to ToHtmlCssColorCode

**Pull request merged**